src/validations/html/slowparse.js

```diff
@@ -124,7 +125,11 @@ class SlowparseValidator extends Validator {
   }
 
   async _getRawErrors() {
-    const {Slowparse} = await System.import('../linters');
+    const Slowparse = await retryingFailedImports(() => import(
+      /* webpackChunkName: 'linters' */
+      '../../util/slowparse',
+    ));
+    console.log(Slowparse);
```

Comment:
@outoftime not sure if you're still online, but if you are—is this an issue you've had before? See the commit message above. And try logging `Slowparse` right before the `export default Slowparse;` line to compare.

I feel like I've spent a cumulative month of my life trying to debug these weird module errors from webpack/Babel and I still don't understand what causes them 😅 

I'll debug some more as soon as I get some more time.